app/src/main/java/com/example/happymeals/recipe/Recipe.java

```diff
@@ -62,6 +62,16 @@ public Recipe( String name, double cookTime, String description, ArrayList< Stri
         this.servings = servings;
     }
 
+    public Recipe(String description) {
```

Comment:
What is this? I am seeing a constructor. Why is there a constructor that only has description??